UnitsNet/UnitDefinitions/RotationalAcceleration.json

```diff
@@ -0,0 +1,32 @@
+﻿{
+    "Name": "RotationalAcceleration",
```

Comment:
Good call following the naming convention of RotationalSpeed, and not calling it AngularAcceleration although I personally would prefer that term.